src/plugins/janus_audiobridge.c

```diff
@@ -5541,6 +5543,7 @@ static json_t *janus_audiobridge_process_synchronous_request(janus_audiobridge_s
 		}
 		/* Return a list of all announcements */
 		json_t *list = json_array();
+#ifdef HAVE_LIBOGG
```

Comment:
I think this should wrap the whole `listannouncements`, or the beginning of it to return a verbose error if libogg isn't available.

Reply:
Checking the code, the latter is what we do for `play_file`, `is_playing` and `stop_playing` already, so I'd follow the same approach.